### PR TITLE
Added Rating and snackbar

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Screenshot (37)
Screenshot (38)

### CODE REVIEW

**Comment by SRUJANA-PENUGONDA13** [comment]
It looks good to me. Below are few suggestions

- Please add links in documentation page so it will be easy for preview while reviewing.
- Try to align content in the code snippet as well.  It seems alignment is lost after copying into corbon.one.sh Before copying into it just copy paste in empty html document and align it using plugins like prettier.

File: Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+:root {
+    --primary-color: #1d4ed8;
+    --text-color: #e5e7eb;
+    --secondary-color: #9ca3af;
+    --bg-color: #e5e7eb;
+    --snackbar-bg-color:#44403c;
+    --snackbar-btn-color:rgb(165, 104, 223);
```
Comment (xii-vi) [inline Snackbar/snackbar.css:7]: Use hex code for color here

File: Snackbar/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <div class="snackbar">
+                <div class="snackbar-text">
+                    Can't send photo. Retry in 5 seconds.
+                </div>
+
+                <div class="snackbar-button">
+                    RETRY
+                </div>
+
+                <div class="snackbar-cross-icon">
```
Comment (xii-vi) [inline Snackbar/snackbar.html:76]: Using cross icon instead of putting X would be better, in my opinion

**Review by xii-vi — APPROVED** [review]
Added Few Comments.
Work on that.
Rest is okay to merge.

File: Snackbar/snackbar.html (hunk 2 above)
Comment (syed-shaida-hussain) [reply Snackbar/snackbar.html:76]: Will work on that. Thank you.

**Pull request merged**